src/server/index.jsx

```diff
@@ -289,14 +289,18 @@ server
       });
 
       try {
-        const { service, isAmp, route, variant } = getRouteProps(
-          routes,
-          urlPath,
-        );
-        const data = await route.getInitialData({
+        const {
+          service,
+          isAmp,
+          route: { pageType, getInitialData },
+          variant,
+        } = getRouteProps(routes, urlPath);
+
+        const data = await getInitialData({
           path: url,
           service,
           variant,
+          pageType,
```

Comment:
I struggled to add a test for this (couldn't get spying working as I wanted), and wondered if we could wait for this instead - https://github.com/bbc/simorgh/pull/7058/files#diff-9d7ac6551ba249f8b551e1ca9306caa4R1329-R1374